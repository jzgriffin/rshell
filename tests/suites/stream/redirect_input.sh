tr a-z A-Z < redirect_input.data
cat <redirect_input.data
rev<redirect_input.data
uniq< redirect_input.data
