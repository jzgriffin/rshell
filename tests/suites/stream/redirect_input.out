FOO
BAR
BAR
BAZ
foo
bar
bar
baz
oof
rab
rab
zab
foo
bar
baz
